charts/datadog/templates/_container-trace-agent.yaml

```diff
@@ -73,6 +73,8 @@
     - name: runtimesocket
       mountPath: {{ template "datadog.dockerOrCriSocketPath" . }}
     {{- end }}
+    - name: dsdsocket
```

Comment:
The `dsdsocket` mount is already defined in line 65. It was moved there in #313.